sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/models/DownloadManifestOptions.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.containers.containerregistry.models;
+
+/**
+ * Options for configuring the download manifest operation.
+ */
+public final class DownloadManifestOptions {
+    private String tag;
+    private String digest;
+
+    private DownloadManifestOptions() { }
+
+    /**
+     * Instantiate the options class with tag.
+     * @param tag The tag associated with the manifest.
+     * @return The DownloadManifestOptions object.
+     */
+    public static DownloadManifestOptions fromTag(String tag) {
+        DownloadManifestOptions options = new DownloadManifestOptions();
```

Comment:
Maybe update the private constructor to take values so that this can be a one-liner and so the fields can be final